scormxblock/scormxblock.py

```diff
@@ -275,6 +278,78 @@ class ScormXBlock(StudioEditableXBlockMixin, ScorableXBlockMixin, XBlock):
     )
     has_author_view = True
 
+    def release_all_external_resources(self):
+        """Called when course/chapter/subsection/unit got removed.
+        """
+        self.discard_scorm_package(remove_scorm_pkg_root=True)
```

Comment:
- When removing a `Chapter` or `Section` ... , we also remove packages from S3 or DFS.